src/main/java/ua/com/juja/microservices/teams/utils/Utils.java

```diff
@@ -1,28 +1,37 @@
 package ua.com.juja.microservices.teams.utils;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-import org.springframework.web.client.HttpClientErrorException;
 
 import java.io.IOException;
 import java.util.Collections;
+
 import ua.com.juja.microservices.teams.exceptions.ApiErrorMessage;
 
 /**
  * @author Ivan Shapovalov
  */
 public class Utils {
-    public static ApiErrorMessage convertToApiError(HttpClientErrorException httpClientErrorException) {
-        ObjectMapper mapper = new ObjectMapper();
-        try {
-            return mapper.readValue(httpClientErrorException.getResponseBodyAsString(), ApiErrorMessage.class);
-        } catch (IOException e) {
-            return new ApiErrorMessage(
-                    500, "BotInternalError",
-                    "I'm, sorry. I cannot parse api error message from remote service :(",
-                    "Cannot parse api error message from remote service",
-                    e.getMessage(),
-                    Collections.singletonList(httpClientErrorException.getMessage())
-            );
+    public static ApiErrorMessage convertToApiError(String message) {
+        int contentExists = message.indexOf("content:");
+        ApiErrorMessage apiError = new ApiErrorMessage(
```

Comment:
I think you should think about some code duplication when you create apiError. 
Maybe you need to extract some method for this case or change logic